api/src/lib/xdomainRoute.js

```diff
@@ -0,0 +1,17 @@
+import config from 'config';
+import { readFileSync } from 'fs';
+import koaRoute from 'koa-route';
+
+const xdomainScript = readFileSync(__dirname + '/../../../node_modules/xdomain/dist/xdomain.min.js');
```

Comment:
We won't interpret it, but only read file for write endpoint response.
It's the solution used by @brikou on the WMS project.
An other solution is to use webpack, perhaps, but it will be time consumer.